src/components/Tags/__stories__/index.stories.tsx

```diff
@@ -0,0 +1,90 @@
+import { Meta, Story } from '@storybook/react'
+import React from 'react'
+import Tags, { TagsProps } from '..'
+
+export default {
+  argTypes: {
```

Comment:
Not using this should work with Tags and to add description in the argstable use TSDoc on the interface members

Reply:
Without argTypes & args you have nothing.

With args, this will show this
<img width="1081" alt="Capture d’écran 2021-11-17 à 17 47 32" src="https://user-images.githubusercontent.com/14060273/142244556-67513ecf-defe-4caf-bbf3-25a1d352b816.png">

With ArgTypes 
<img width="810" alt="Capture d’écran 2021-11-17 à 17 54 17" src="https://user-images.githubusercontent.com/14060273/142245587-f7fb88ce-afde-4144-97ce-a378cd9a0c34.png">